### DISCUSSION TITLE
F12报错js，网上的教程看不懂，不清楚该怎么解决。

### DISCUSSION THREAD

**Discussion by Yumemiiiii** [hexojs/hexo#5023] [Question / Answer] [answered]
我从来没有下载过有关js的文件，编程语言也不太懂，谢谢谢谢
![f179be19d3e58194e2f346e4054701a](https://user-images.githubusercontent.com/107329735/181194795-c4b0e5fc-63a5-4d71-bde7-cc7f94d10ac5.png)

Uncaught ReferenceError: $ is not defined
    at main.js:14:2
    at main.js:35:3
index.js:9 
  APlayer v1.10.1 af84efb  http://aplayer.js.org 
Meting.min.js:1 
  MetingJS v1.2.0  https://github.com/metowolf/MetingJS

**Comment by Yumemiiiii** [comment]
script的src引用路径是什么东西啊（T_T
我现在还在学python，真的一点也看不懂

**Comment by stevenjoezhang** [comment]
看上去是 jQuery 没有加载。可以提供一下你的网站链接，这样大家可以帮忙看看。

**Reply by Yumemiiiii** [reply]
https://yumemiiiii.github.io/
之前没有看到，谢谢您

**Reply by stevenjoezhang** [reply, marked as answer]
<img width="671" alt="截屏2022-08-03 13 25 15" src="https://user-images.githubusercontent.com/16272760/182530688-c424cefc-308b-4720-a812-2a879445b796.png">

这段报错的代码是你自己写的吗？我没有在主题的源码里找到

**Reply by Yumemiiiii** [reply]
我记忆中不是我自己写的，因为根本看不懂

**Reply by stevenjoezhang** [reply]
尽量避免在自己的网站中加载来源不明的代码。
